node/Makefile

```diff
@@ -401,12 +400,6 @@ kind-k8st-run-test: .calico_test.created $(KIND_KUBECONFIG)
 .PHONY: kind-k8st-cleanup
 kind-k8st-cleanup: kind-cluster-destroy
 
-# Needed for Semaphore CI (where disk space is a real issue during k8s-test)
```

Comment:
Not needed and not referenced. We are fine with semaphore disk space after the calico/go-build image size reduction work.